src/common/translations/pt.json

```diff
@@ -222,12 +222,8 @@
       "addCustomToken": {
         "title": "Adicionar ativos personalizados",
         "address": "Endereço do contrato do ativo",
-<<<<<<< HEAD
-        "mainnet": "Rede principal"
-=======
-        "mainnet": "Mainnet",
-        "testnet": "Testnet"
->>>>>>> develop
+        "mainnet": "Rede principal",
+        "testnet": "Rede de teste"
```

Comment:
```suggestion
        "testnet": "Rede de testes"
```

Reply:
Portuguese Brazilian?
The file is pt.json.